frontend/public/components/utils/details-page.tsx

```diff
@@ -118,11 +118,11 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({
               onClick={Kebab.factory.ModifyTolerations(model, resource).callback}
               variant="link"
             >
-              {pluralize(_.size(tolerations), 'Toleration')}
+              {t('details-page~{{count}} toleration', { count: _.size(tolerations) })}
               <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
             </Button>
           ) : (
-            pluralize(_.size(tolerations), 'Toleration')
+            t('details-page~Toleration', { count: _.size(tolerations) })
```

Comment:
```suggestion
            t('details-page~{{count}} Toleration', { count: _.size(tolerations) })
```

Reply:
Mine should output "Toleration" or "Tolerations." If I do `{{count}} Toleration`, it'll be "2 Tolerations," etc.

Reply:
Good catch; thanks!